include "atoms/tenfour.thrift"

union ContentAtom {
  1: tenfour.TenfourQuizBuilderAtom tenfour
}
